src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -12,14 +12,12 @@ ARG ARCH="amd64"
 ARG YQ_ARCH="amd64"
 # PROTOC_ARCH="x86_64" | "s390_64"
 ARG PROTOC_ARCH="x86_64"
-ARG GO_VERSION="1.21.12"
-ARG PROTOC_VERSION="3.15.0"
-ARG RUST_VERSION="1.75.0"
-ARG YQ_VERSION="v4.35.1"
-ARG PACKER_VERSION="v1.9.4"
-# amd64: YQ_CHECKSUM="sha256:bd695a6513f1196aeda17b174a15e9c351843fb1cef5f9be0af170f2dd744f08"
-# s390x: YQ_CHECKSUM="sha256:4e6324d08630e7df733894a11830412a43703682d65a76f1fc925aac08268a45"
-ARG YQ_CHECKSUM="sha256:bd695a6513f1196aeda17b174a15e9c351843fb1cef5f9be0af170f2dd744f08"
+ARG GO_VERSION
+ARG PROTOC_VERSION
+ARG RUST_VERSION
+ARG YQ_VERSION
+ARG PACKER_VERSION
```

Comment:
do you need ENV setting for PACKER_VERSION later in the code?

Reply:
I'm not sure if it's strictly required as that seems to be GHA related and I don't see us ever building RHEL in the open-source pipelines, but I can add for consistency